### ISSUE TITLE
Add support for "fix" option to config file

### ISSUE THREAD

**Issue by baolsen** [terraform-linters/tflint#2369]
### Summary

tflint recently added a `--fix` CLI application parameter. 

Support for the same thing should be added to the `.tflint.hcl` config file, for consistency.

I prefer to have a single way to configure the application, but at the moment I need to provide both a config file as well as flags. 

Here is an example of how it might look
```hcl
config {
  fix = true
}
```

### Command

tflint --args=--config=__GIT_WORKING_DIR__/.tflint.hcl

### Terraform Configuration

```terraform
Not sure it matters for this report
```

### TFLint Configuration

```HCL
config {
  # disabled_by_default = true
  fix = true
}

rule "terraform_deprecated_interpolation" {
  enabled = true
}

rule "terraform_deprecated_index" {
  enabled = true
}

rule "terraform_unused_declarations" {
  enabled = true
}

rule "terraform_comment_syntax" {
  enabled = true
}

rule "terraform_documented_outputs" {
  enabled = true
}

rule "terraform_documented_variables" {
  enabled = true
}

rule "terraform_typed_variables" {
  enabled = true
}

rule "terraform_module_pinned_source" {
  enabled = true
}

rule "terraform_naming_convention" {
  enabled = true
}

rule "terraform_required_version" {
  enabled = true
}

rule "terraform_required_providers" {
  enabled = true
}

rule "terraform_standard_module_structure" {
  enabled = true
}

rule "terraform_workspace_remote" {
  enabled = true
}
```

### Output

```Shell
Command 'tflint --init' failed:
Failed to load TFLint config; /home/myuser/myrepo//.tflint.hcl:3,3-6: Unsupported argument; An argument named "fix" is not expected here.:

Error: Unsupported argument
```

### TFLint Version

0.58.1

### Operating System

- [x] Linux
- [ ] macOS
- [x] Windows
